configs/httpd/dist-git/lookaside-upload.conf.example

```diff
@@ -15,98 +13,58 @@ SSLCryptoDevice builtin
 # SSL host
 <VirtualHost _default_:443>
     # This alias must come before the /repo/ one to avoid being overridden.
-    ScriptAlias /repo/pkgs/upload.cgi /var/lib/dist-git/web/upload.cgi
-
-    Alias /repo/ /var/lib/dist-git/cache/lookaside/
+    ScriptAlias /repo/pkgs/upload.cgi /srv/web/upload.cgi
 
-    #ServerName pkgs.fedoraproject.org
-    #ServerAdmin webmaster@fedoraproject.org
+    Alias /repo/ /srv/cache/lookaside/
+    ServerName pkgs.example.org
+    ServerAdmin webmaster@fedoraproject.org
 
     SSLEngine on
 
-    # Server Certificate:
-    # Point SSLCertificateFile at a PEM encoded certificate.  If
-    # the certificate is encrypted, then you will be prompted for a
-    # pass phrase.  Note that a kill -HUP will prompt again.  A new
-    # certificate can be generated using the genkey(1) command.
-    SSLCertificateFile /etc/pki/tls/certs/localhost.crt
-
-    # Server Private Key:
-    # If the key is not combined with the certificate, use this
-    # directive to point at the key file.  Keep in mind that if
-    # you've both a RSA and a DSA private key you can configure
-    # both in parallel (to also allow the use of DSA ciphers, etc.)
-    SSLCertificateKeyFile /etc/pki/tls/private/localhost.key
-
-    # Certificate Authority (CA):
-    # Set the CA certificate verification path where to find CA
-    # certificates for client authentication or alternatively one
-    # huge file containing all of them (file must be PEM encoded)
-    SSLCACertificateFile /etc/pki/tls/certs/ca-bundle.crt
-
-    # Certificate Revocation Lists (CRL):
-    # Such a file is simply the concatenation of the various
-    # PEM-encoded CRL files, in order of preference.
-    # These are used to revoke the client certificate on 
-    # Client Authentication.
-    SSLCARevocationFile  /etc/pki/tls/crl.pem
+    SSLCertificateFile    /etc/pki/tls/certs/pkgs.example.org.pem
+    SSLCertificateKeyFile /etc/pki/tls/certs/pkgs.example.org.pem
+    SSLCACertificateFile  /etc/pki/tls/certs/ca-bundle.crt
+    #SSLCARevocationFile   /etc/pki/tls/certs/crl.pem
 
-    SSLCipherSuite RSA:!EXPORT:!DH:!LOW:!NULL:+MEDIUM:+HIGH
+    SSLProtocol -All +TLSv1 +TLSv1.1 +TLSv1.2
+    SSLCipherSuite ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!3DES:!MD5:!PSK
```

Comment:
As far as I understand it;  when it is reasonable possible Fedora's services should use the `PROFILE=SYSTEM` cipher suite specification ... rather than hard-code something service-specific (so user/distribution can change the default trusted ciphers system-wide, e.g.)